README.md

```diff
@@ -128,10 +122,25 @@ To add `SwiftTweaks` to your application, add it to your `Cartfile`:
 github "Khan/SwiftTweaks" ~> 1.0
 ```
 
+In addition, add `-DDEBUG` to **Other Swift Flags** in your project's Build Settings for your *Debug* configuration.
```

Comment:
By default, the value doesn't exist in Xcode with Swift now (unlike with Objective-C)